clipmenu

```diff
@@ -30,14 +30,26 @@ if [[ "$CM_LAUNCHER" == rofi ]]; then
     set -- -dmenu "$@"
 fi
 
-# It's okay to hardcode `-l 8` here as a sensible default without checking
-# whether `-l` is also in "$@", because the way that dmenu works allows a later
-# argument to override an earlier one. That is, if the user passes in `-l`, our
-# one will be ignored.
-chosen_line=$(
-    cat "$cache_file_prefix"_* /dev/null | LC_ALL=C sort -rnk 1 |
-        cut -d' ' -f2- | awk '!seen[$0]++' | "$CM_LAUNCHER" -l 8 "$@"
-)
+list_clips() {
+    cat "$cache_file_prefix"_* /dev/null | LC_ALL=C sort -rnk 1 | cut -d' ' -f2- | awk '!seen[$0]++'
+}
+
+if [[ "$CM_LAUNCHER" == rofi-script ]]; then
+    if [[ -z "$@" ]]; then
+        list_clips
+        exit
+    else
+        chosen_line="$@"
```

Comment:
If this is how rofi inputs the new selection, it's unlikely to work very well with our policy to preserve whitespace accurately. Or maybe it's really just `$1`? I'll have a look myself this weekend.

Reply:
It's really just `$1`, I can change this.